packages/federatorai/overlay/questions.yaml

```diff
@@ -0,0 +1,79 @@
+questions:
+#image configurations
+- variable: defaultImage
+  default: true
+  description: "Use default Federator.ai image or specify a custom one"
+  label: Use Default Federator.ai Image
+  type: boolean
+  show_subquestion_if: false
+  group: "Container Images"
+  subquestions:
+  - variable: image.repository
+    default: "quay.io/prophetstor/federatorai-operator-ubi"
+    description: "Federator.ai Operator image name"
+    type: string
+    label: Federator.ai Operator Image Name
+  - variable: image.tag
+    default: "v4.5.1-ga"
+    description: "Federator.ai Operator image tag"
+    type: string
+    label: Federator.ai Operator Image Tag
+#service configurations
+- variable: federatorai.imageLocation
+  default: "quay.io/prophetstor"
+  description: "Service containers image location"
+  type: string
+  required: true
+  label: Federator.ai imageLocation
+- variable: federatorai.version
+  default: "v4.5.1-ga"
+  description: "Service containers version"
+  type: string
+  required: true
+  label: Service Containers Image Tag
+- variable: services.dashboardFrontend.nodePort
+  required: true
+  default: "31012"
+  description: "The port where the Federator.ai Dashboard listens to"
+  type: string
+  label: Federator.ai Dashboard Port
+- variable: services.rest.nodePort
+  required: true
+  default: "31011"
+  description: "The port where the Federator.ai REST listens to"
+  type: string
+  label: Federator.ai REST Port
+- variable: federatorai.persistence.enabled
+  default: true
```

Comment:
The chart installs corrently, and overall looks ready to merge. However, these aren't showing up in the UI because they are missing the `group: <group-name>`. Please update these and I'll approve/merge.

Reply:
Add `group: <group-name>`.